pipeline/src/main/java/io/datacater/Pipeline.java

```diff
@@ -0,0 +1,114 @@
+package io.datacater;
+
+import io.datacater.exceptions.TransformationException;
+import io.smallrye.common.annotation.Blocking;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.smallrye.reactive.messaging.kafka.Record;
+
+import java.util.UUID;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.core.buffer.Buffer;
+import io.vertx.mutiny.ext.web.client.HttpRequest;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+import io.vertx.mutiny.ext.web.client.WebClient;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.eclipse.microprofile.reactive.messaging.*;
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.RestResponse;
+
+@ApplicationScoped
+public class Pipeline {
+  private static final Logger LOGGER = Logger.getLogger(Pipeline.class);
+  private String host;
+  private Integer port;
+
+  WebClient client;
+
+  @Inject
+  void VertxResource(Vertx vertx) {
+    this.client = WebClient.create(vertx);
+  }
+
+  @Inject
+  @Channel(PipelineConfig.STREAM_OUT)
+  Emitter<Record<JsonObject, JsonObject>> producer;
+
+  @Incoming(PipelineConfig.STREAM_IN)
+  @Blocking
+  @Acknowledgment(Acknowledgment.Strategy.POST_PROCESSING)
+  public void processUUID(ConsumerRecords<JsonObject, JsonObject> messages) {
+    // This is deliberately blocking
+    handleMessages(messages);
+  }
+
+  private void handleMessages(ConsumerRecords<JsonObject, JsonObject> messages) {
+    HttpRequest<Buffer> request = client.post(getPort(), getHost(), PipelineConfig.ENDPOINT)
```

Comment:
One thing that I just noticed: Some Python transforms might need longer than expected, requiring us to abort the evaluation of the pipeline spec in the runner.

I believe we should add a timeout here.  A reasonable default might be something like 60 seconds, but we should definitely make it configurable.

Reply:
added ✅

Reply:
Great, thanks a lot. LGTM 💯